game-core/src/test/java/games/strategy/triplea/delegate/battle/MustFightBattleExecutablesTest.java

```diff
@@ -0,0 +1,1671 @@
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnit;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitAirTransport;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitCanEvade;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitDestroyer;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitIsAir;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.givenUnitTransport;
+import static games.strategy.triplea.delegate.battle.steps.BattleStepsTest.newUnitAndAttachment;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.lessThan;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.GamePlayer;
+import games.strategy.engine.data.MutableProperty;
+import games.strategy.engine.data.RelationshipTracker;
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitCollection;
+import games.strategy.engine.data.UnitType;
+import games.strategy.engine.data.changefactory.ChangeFactory;
+import games.strategy.engine.data.properties.GameProperties;
+import games.strategy.engine.delegate.IDelegateBridge;
+import games.strategy.triplea.Constants;
+import games.strategy.triplea.attachments.TechAttachment;
+import games.strategy.triplea.attachments.UnitAttachment;
+import games.strategy.triplea.delegate.IExecutable;
+import java.math.BigDecimal;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Set;
+import junit.framework.AssertionFailedError;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.java.collections.IntegerMap;
+import org.triplea.sound.ISound;
+import org.triplea.test.common.Integration;
+import org.triplea.util.Tuple;
+
+@ExtendWith(MockitoExtension.class)
+@Integration
```

Comment:
I marked this `@Integration` because I didn't want to put these tests in the original MustFightBattleTest.java file.  The checks complained about having a test class that isn't connected to a non-test class unless I put `@Integration` on it.  I can move the tests to the other file if this is a problem.

Once I've refactored steps, these tests will be moved to the individual step test file so it is just temporary.

Reply:
`@SuppressWarnings("UnmatchedTest")` Should do the trick directly